console/src/socket_client.rs

```diff
@@ -0,0 +1,14 @@
+use std::io::prelude::*;
+use std::net::TcpStream;
+
+pub fn send_data_to_server(data: &str, server_address: &str) {
+    match TcpStream::connect(server_address) {
```

Comment:
Please obtain `server_address` from an environmental variable instead of a parameter.

https://en.wikibooks.org/wiki/Guide_to_Unix/Environment_Variables